client/readme.md

```diff
@@ -0,0 +1,37 @@
+# Elm Frontend Readme
+
+The Lunar Rocks frontend is an Elm 0.19 application.
+
+==================================================
+
+## Setup
+
+Install npm packages from the `client` directory.
+```
+npm install
+```
+
+Install [Elm](https://guide.elm-lang.org/install.html) and [elm-Live](https://github.com/wking-io/elm-live), and [uglify-js](https://www.npmjs.com/package/uglify-js).
+
+## Develop 
+
+Start the elm app from the `client` directory.
+
+```
+bash develop.sh
```

Comment:
Are the `bash` part of these commands really necessary? why not just change the shebang in each script to `#!/bin/bash`?